step-certificates/examples/certificate_authority_single_instance/values.yml.tpl

```diff
@@ -0,0 +1,130 @@
+kind: StatefulSet
+replicaCount: 1
+autocert:
+  enabled: false
+bootstrap:
+  enabled: false
+ca:
+  db:
+      enabled: true
+      persistent: false
+inject:
+  enabled: true
+  config:
+    files:
+      ca.json:
+        root: /home/step/certs/root_ca.crt
+        federateRoots: []
+        crt: /home/step/certs/intermediate_ca.crt
+        key: /home/step/secrets/intermediate_ca_key
+        address: 0.0.0.0:9000
+        dnsNames: ${CA_DNS_NAMES}
+        logger:
+          format: json
+        db:
+          type: badger
+          dataSource: /home/step/db
+        authority:
+          claims:
+            minTLSCertDuration: 5m
+            maxTLSCertDuration: 24h
+            defaultTLSCertDuration: 24h
+            disableRenewal: false
+            minHostSSHCertDuration: 5m
+            maxHostSSHCertDuration: 1680h
+            defaultHostSSHCertDuration: 720h
+            minUserSSHCertDuration: 5m
+            maxUserSSHCertDuration: 24h
+            defaultUserSSHCertDuration: 24h
+          provisioners:
+            - type: ACME
+              name: acme
+              forceCN: true
+              claims: {}
+            - type: SSHPOP
+              name: sshpop
+              claims:
+                enableSSHCA: true
+            - type: JWK
+              name: ansible_automation_token
+              key: 
+                alg: "${JWK_PROVISIONER_CRT_ALG}"
+                crv: "${JWK_PROVISIONER_CRT_CRV}"
+                kid: "${JWK_PROVISIONER_CRT_KID}"
+                kty: "${JWK_PROVISIONER_CRT_KTY}"
+                use: "${JWK_PROVISIONER_CRT_USE}"
+                x: "${JWK_PROVISIONER_CRT_X}"
+                'y': "${JWK_PROVISIONER_CRT_Y}"
+              encryptedKey: "${JWK_PROVISIONER_KEY}"
+              claims:
+                enableSSHCA: true
+        tls:
+          cipherSuites:
+            - TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
+            - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+            - TLS_AES_128_GCM_SHA256
+          minVersion: 1.2
+          maxVersion: 1.3
+          renegotiation: false
+      defaults.json:
+        ca-url: https://mysteprelease-step-certificates.default.svc.cluster.local
+        ca-config: /home/step/config/ca.json
+        fingerprint: fingerprint
```

Comment:
We should get this from a template too, and fill that variable like this:
```shell
ROOT_CA_FINGERPRINT=$(step certificate fingerprint root_ca.crt)
```

Reply:
added this template value as well as templated the ca-urll